setup.py

```diff
@@ -34,6 +34,7 @@
     install_requires=[
         'click',
         'click-default-group',
+        'natsort',
         'tabulate'
```

Comment:
Unfortunately, we can't list sswsdk as a dependency here. Since we compile sonic-utilities into a .deb package, stdeb converts all Python dependencies here into Debian dependencies, thus it looks for installed .deb packages named "python-<python_package_name>. However, since we install sswsdk directly as a Python Wheel, it does not show up as an installed .deb package, so the  sonic-utilities install will fail. We may want to consider unifying our package creation process.